packages/roosterjs-editor-core/lib/corePlugins/DOMEventPlugin.ts

```diff
@@ -29,7 +30,10 @@ export default class DOMEventPlugin implements EditorPlugin {
     private cachedPosition: NodePosition;
     private cachedFormatState: PendableFormatState;
 
-    constructor(private disableRestoreSelectionOnFocus: boolean) {}
+    constructor(
+        private disableRestoreSelectionOnFocus: boolean,
+        public readonly scrollContainer?: HTMLElement
```

Comment:
Should this param be required? Will DOMEventPlugin be created somewhere other than in `createEditorCore`?